ironic/drivers/modules/drac/common.py

```diff
@@ -115,3 +115,8 @@ def get_drac_client(node):
                                     driver_info['drac_protocol'])
 
     return client
+
+
+def get_reboot_options():
```

Comment:
I think this change may be showing up because I resynced my fork with Ironic.  If so, can you undo this change?

Reply:
Sure will undo this changes.